modules/database/variables.tf

```
@@ -139,3 +139,14 @@ variable "tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "auto_pause_delay_in_minutes" {
+  description = "Time in minutes after which database is automatically paused. Only settable for Serverless databases."
```

Update description to match usual phrasing:

```suggestion
  description = "Time in minutes after which this SQL database is automatically paused. Value must be greater than 0 if using a SKU in the serverless compute tier."
```